copy/multiple.go

```diff
@@ -12,11 +12,46 @@ import (
 	internalManifest "github.com/containers/image/v5/internal/manifest"
 	"github.com/containers/image/v5/manifest"
 	"github.com/containers/image/v5/signature"
+	digest "github.com/opencontainers/go-digest"
 	imgspecv1 "github.com/opencontainers/image-spec/specs-go/v1"
 	"github.com/sirupsen/logrus"
 	"golang.org/x/exp/slices"
 )
 
+type CopyOperation int
+
+const (
+	Ignore CopyOperation = iota + 1
+	CopyWithoutModification
+	//TODO: following states must be implemented
+	//CopyWithModification
+	//CopyAndReplicate
+)
+
+type instanceCopy struct {
+	digest digest.Digest
+	op     CopyOperation
+}
+
+// instancesToCopy returns the list of digests which actually need to be copied to the new manifest list, ignoring the ones which are not needed.
+func (c *copier) instancesToCopy(options *Options, list internalManifest.List) (instances []instanceCopy, copyCount int) {
```

Comment:
(Absolutely non-blocking: I’d prefer not to use named return values; they tend to result in non-obvious bugs caused just by a missing `:` in `:=` somewhere, especially with `err` and `defer` involved.

I expect this will all change anyway…)